backend/timed/tracking/tests/test_attendance.py

```diff
@@ -127,3 +127,24 @@ def test_attendance_to_before_from(internal_employee_client, attendance_factory)
     assert json["errors"][0]["detail"] == (
         "An attendance may not end before it starts."
     )
+
+
+def test_attendance_can_end_at_midnight(internal_employee_client, attendance_factory):
+    """Test that 0:00 is a valid to_time."""
+    attendance = attendance_factory(
+        user=internal_employee_client.user, from_time=time(7, 30), to_time=time(8, 30)
+    )
+
+    data = {
+        "data": {
+            "type": "attendances",
+            "id": attendance.id,
+            "attributes": {"to-time": "00:00"},
```

Comment:
so from_time = 07:00, to_time = 00:00 - what's the expected duration? Might be worth testing as well (unless it's already tested elsewhere)

Reply:
the expected duration should be 17h (24 - 7), but AFAICT all the duration things for attendances are done in the frontend